src/core/settings.py

```diff
@@ -265,8 +287,16 @@
 EMAIL_USE_TLS = bool(os.getenv('EMAIL_USE_TLS', True))
```

Comment:
I know this isn't a change, but I see a flaw in the use of `bool` on `os.getenv(...)`. If the env is set to any value except for an empty string, it will evaluate to true. So in most cases except for empty strings, this will always evaluate to true. We want this to be true anyway, but still.